src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -91,6 +96,10 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
This should be called after the EnrollCommand, because once updateFilteredPersonList is called, the list will be updated to the whole list, so let's say initially Ben is at index 0 and i want to edit the name of Ben, so after UnEnrollCommand to remove every participation object, then updateFilteredPersonList is called, so now the list includes everyone, and Ben might no longer be at index 0, as it could be Alice or something now. So calling EnrollCommand on index 0 will be executing the command on Alice, which i think shouldn't be the behaviour

Reply:
Make sense, good catch